### ISSUE TITLE
Configure a site does not meet contrast guidelines

### ISSUE THREAD

**Issue by Theresa-o** [wagtail/wagtail#9569]
### Issue Summary
The Configure a site link has an insufficient color contrast of 4.25 between the foreground color: #008080, background color: #e3f5fc

### Steps to Reproduce

1. Go to admin user interface
2. Click on Pages
3. You would see Configure a site

Expected result: The colours on this button should meet contrast requirements ratio of 4.5:1

-   I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: (yes)

![wagfirstissuecolorcontrastissue](https://user-images.githubusercontent.com/62052443/199395569-56a41ce1-afe4-4ca2-a40d-7dfca7b658fc.PNG)

Recommended solution
The SCSS is in the file client/scss/components/_help-block.scss

Change the foreground color to use the colour #007A7A

### Technical details

-   Python version: Run `python --version`.
-   Django version: Look in your requirements.txt, or run `pip show django | grep Version`.
-   Wagtail version: Look at the bottom of the Settings menu in the Wagtail admin, or run `pip show wagtail | grep Version:`.
-   Browser version: You can use https://www.whatsmybrowser.org/ to find this out.

**Comment by Theresa-o** [comment]
I'm working on this issue

**Comment by lb-** [comment]
Confirming this is an issue.

<img width="1351" alt="Screen Shot 2022-11-07 at 7 42 46 pm" src="https://user-images.githubusercontent.com/1396140/200278563-c80f5f88-f67a-42d3-9493-bb8081c8fa28.png">

**Comment by thibaudcolas** [comment]
For future reference – just backported this to `stable/4.1.x`, as the dashboard’s upgrade notification banner’s link is making our [accessibility checks fail](https://app.circleci.com/pipelines/github/wagtail/wagtail/12568/workflows/7952a7d3-8abf-4da6-8f69-8a5fbe8f9e6d/jobs/36054) 😅
